modules/RTC/BandwidthLimiter.js

```diff
@@ -0,0 +1,80 @@
+/* global __filename */
+
+import transform from 'sdp-transform';
+import { getLogger } from 'jitsi-meet-logger';
+
+const logger = getLogger(__filename);
+
+/**
+ * This will save a bandwidth limit (per mline) that should be used
+ * and then, when given an SDP, will enforce that limit.
+ * Note that this will affect *outgoing* bandwidth usage, but the SDP
+ * it must modify to implement that is the *remote* description
+ */
+export default class BandwidthLimiter {
+
+    /**
+     * Create a new BandwidthLimiter
+     */
+    constructor() {
+        /**
+         * Map of mline media type to a bandwidth
+         * limit (in kbps).  Any mlines present in this
+         * map will have the associated bandwidth limit
+         * (or 'null' for no limit) enforced, meaning that
+         * it will potentially overwrite a limit
+         * already set in the given sdp.  However, if an
+         * mline is NOT present in this map, any limit
+         * in the given sdp will not be touched.
+         */
+        this._bandwidthLimits = {};
+    }
+
+    /**
+     * Set a bandwidth limit for given mline.  If
+     * limitKbps is null, the limit will be removd
+     * @param mediaType the mline media type to set the
```

Comment:
could you please describe the types of each `@param` ? we try to do that in jitsi-meet. This applies to all code added in this PR

Reply:
fixed